### ISSUE TITLE
Send Video/Image Files to website

### ISSUE THREAD

**Issue by ayoadejoe** [sborsay/Arduino_Wireless#1]
Hello, your code works. The problem is that my assignment involves sending files to a website. I know how to use it send files to my local browser and that is possible because the esp acts as the webserver on my network. The challenge is how to make it function as a client and write files to the site database/Storage preferably through PHP.
Is there a way to go about this?

**Comment by sborsay** [comment]
yes you can accomplish this in Node.JS or PHP but I don't current;y have code for doing that.  I'm sure someone has PHP code to get it into a Mongo DB or SQL varient
